_saas-integrations/kustomer/v1/kustomer-v1.md

```diff
@@ -0,0 +1,133 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Kustomer
+permalink: /integrations/saas/kustomer
+keywords: kustomer, integration, schema, etl kustomer, kustomer etl, kustomer schema
+layout: singer
+# input: false
+
+key: "kustomer-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "kustomer"
+display_name: "Kustomer"
+
+singer: true
+status-url: ""
+
+tap-name: "kustomer"
```

Comment:
Remember that this is the **display** name of the tap. So it would be `Kustomer`.